### ISSUE TITLE
List available updates on the Dashbord

### ISSUE THREAD

**Issue by da2x** [martinpitt/cockpit-os-updates#2]
It would be great to see which systems are up-to-date and which need attention right from the systems list on the Cockpit dashboard (localhost:9090/dashboard).

**Comment by martinpitt** [comment]
Yes, agreed, and this already came up during design discussions. This is a bit tricky though (state and UI changes across Cockpit pages), so it won't be in version 1.
